README.md

```diff
@@ -80,10 +80,48 @@ You can also set the gas type and flow rate / pressure.
 
 ```python
 flow_controller.set_gas('N2')
+flow_controller.set_gas(8)         # Optionally set a gas by it's number; find the full gas table on page 52 of the Alicat manual.
 flow_controller.set_flow_rate(1.0)
 flow_controller.set_pressure(20)
 ```
 
+For firmwave 5v and greater, create and set gas mixes using COMPOSER software loaded into the device. Mixes can contain up to five gases, and are stored in gas indices 236-255. 
```

Comment:
ware, not wave.

Is there a way to read the firmware version through the serial commands? We could use this on init to set features and raise errors if someone tries to use an unsupported function.

Reply:
Nevermind, you already do this and it's great.